spec/test.go

```diff
@@ -27,7 +28,9 @@ func (t *Test) AddError(args Args, response *Response) error {
 }
 
 func main() {
+	socket_path := os.Args[1]
```

Comment:
Mainly because its very explicit to use a flag and because defaults are trivial to specify.